caddy/caddymain/run.go

```diff
@@ -97,6 +97,12 @@ func Run() {
 		mustLogFatalf(err.Error())
 	}
 
+	// Execute Startup Hooks
+	err = caddy.StartupHooks(serverType)
+	if err != nil {
+		mustLogFatalf(err.Error())
+	}
```

Comment:
Change to `"%v", err` just to be safe (I know, I know, we don't do that anywhere else in this file, I just thought of it.)